decoimpact/data/parsers/parser_classification_rule.py

```diff
@@ -54,3 +61,223 @@ def parse_dict(self, dictionary: Dict[str, Any], logger: ILogger) -> IRuleData:
             output_variable_name,
             description,
         )
+
+    def _validate_table_coverage(self, crit_table, logger: ILogger):
+        msgs = []
+        criteria_table = crit_table.copy()
+        del criteria_table["output"]
+
+        def check_coverage(criteria_table, conditions={}, unique=True):
```

Comment:
if I understand correctly, this is done via a recursive function to sort all criteria (and then validate them); correct? 
Maybe rename then or add this to the description?